charts/loki-simple-scalable/values.yaml

```diff
@@ -59,68 +59,53 @@ loki:
   existingSecretForConfig: ""
   # -- Config file contents for Loki
   # @default -- See values.yaml
-  config: |
+  config:
     auth_enabled: false
 
     server:
       http_listen_port: 3100
 
     memberlist:
       join_members:
-        - {{ include "loki.fullname" . }}-memberlist
+        - "{{ include \"loki.fullname\" . }}-memberlist"
 
-    {{- if .Values.loki.commonConfig}}
     common:
-    {{- toYaml .Values.loki.commonConfig | nindent 2}}
```

Comment:
don't want to lose this either

Reply:
this one have dependencies and the value has been copied over. won't fix.

Reply:
again I don't understand. what dependencies prevent us from having a common config section?